src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -345,8 +353,15 @@ private void OpenIndexes()
                     if (int.TryParse(indexDirectory.Name, out indexId) == false)
                         continue;
 
-                    var index = Index.Open(indexId, _documentDatabase);
-                    _indexes.Add(index);
+                    try
+                    {
+                        var index = Index.Open(indexId, _documentDatabase);
+                        _indexes.Add(index);
+                    }
+                    catch (Exception e)
+                    {
+                        Log.ErrorException($"Could not open index with id {indexId}", e);
```

Comment:
Otherwise, we are just going to "disappear" the index, and if the log isn't on, we will miss that.